components/TextFields/examples/TextFieldOutlinedExample.m

```diff
@@ -38,9 +43,26 @@ - (void)dealloc {
   [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
+- (void)styleTextInputController:(id<MDCTextInputController>)controller {
+  [MDCOutlinedTextFieldColorThemer applySemanticColorScheme:self.colorScheme
+                                      toTextInputController:controller];
+  [MDCTextFieldTypographyThemer applyTypographyScheme:self.typographyScheme
+                                toTextInputController:controller];
+  [MDCTextFieldTypographyThemer applyTypographyScheme:self.typographyScheme
+                                          toTextInput:controller.textInput];
+}
+
 - (void)viewDidLoad {
   [super viewDidLoad];
-  self.view.backgroundColor = [UIColor whiteColor];
+
+  if (!self.colorScheme) {
```

Comment:
`viewDidLoad` is guaranteed to be called once. I think you can drop these null checks.

Reply:
It's in case the schemes are injected by the Catalog, we don't want to overwrite those with the defaults.